systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaConnectorUtils.java

```diff
@@ -44,9 +44,14 @@ public static void waitForConnectorStability(String connectorName, String connec
     }
 
     public static void waitForConnectorStatus(String name, String state) {
+        waitForConnectorStatus(name, state, () -> { });
+    }
+
+    public static void waitForConnectorStatus(String name, String state, Runnable onTimeout) {
```

Comment:
For this and all the other `waitFor*()` methods, when and why _don't_ you want to have the debug logging on timeout? In other words, what are the callers of `waitForConnectorStatus(String name, String state)` doing that means they're not needing the debug logging?

Reply:
That's really interesting idea, thanks :)